integration-test/src/main/java/com/sequenceiq/it/cloudbreak/UpgradeTests.java

```diff
@@ -16,9 +17,15 @@
     private CloudProvider cloudProvider;
 
     public UpgradeTests() {
-        this.cloudProvider = new GcpCloudProvider();
+        this.cloudProvider = new GcpCloudProvider(getTestParameter());
     }
 
+    public UpgradeTests(CloudProvider cp, TestParameter tp) {
+        this.cloudProvider = new GcpCloudProvider(getTestParameter());
```

Comment:
Wouldn't we like to use the "cp" parameter inside of the constructor instead of creating a GCP based provider every time?